v2/brigadier/src/events.ts

```diff
@@ -1,26 +1,58 @@
 import { Worker } from "./workers"
 import { Project } from "./projects"
 
+/**
+ * An event received by Brigade that resulted in triggering an event handler.
+ */
```

Comment:
My initial reaction to the term "handler" was that it was inconsistent with how we talk about Brigade. We might use the term "handler" when defining what a "worker" is (a containerized event handler), but otherwise we haven't used that term much.

Then I realized that "handler" here is more in reference to the _Brigadier_ programming model. Since we do things like `events.on("someEvent", someFunction)`............. yup....... that's a handler alright.

So contextually, this makes perfect sense. 👍